frontend/src/App.tsx

```diff
@@ -871,4 +874,4 @@ class App extends PureComponent<Props, State> {
   }
 }
 
-export default App
+export default withScreencast(App)
```

Comment:
see this [comment](https://github.com/streamlit/streamlit/pull/929#issuecomment-574784415)

We are decorating App with screencast generating a new component, App does not needs to know about screencast, this will improve composition, It's a very common react pattern